doc/whats_new/v0.23.rst

```diff
@@ -290,6 +290,10 @@ Changelog
   of strictly inferior for maximum of `absgrad` and `tol` in `utils.optimize._newton_cg`.
   :pr:`16266` by :user:`Rushabh Vasani <rushabh-v>`.
 
+- |Fix| |Efficiency| :class:`linear_model.ARDRegression` is more stable and
+  much faster. It can now scale to hundreds of thousands of samples.
```

Comment:
you could also mention https://github.com/scikit-learn/scikit-learn/pull/16849/files#r403760030 here maybe?